modules/nf-core/kraken2/kraken2/nextflow.config

```diff
@@ -0,0 +1,15 @@
+if (!params.skip_alignment && !params.skip_qc) {
+    if (params.contaminant_screening in ['kraken2', 'kraken2_bracken']) {
+        process {
+            withName: 'KRAKEN2' {
+                // See https://doi.org/10.1080/19490976.2024.2323235 for the confidence level
+                // and https://doi.org/10.1038/s41596-022-00738-y for the minimum-hit-groups
+                ext.args = "--confidence 0.05 --minimum-hit-groups 3"
+                publishDir = [
+                    path: { "${params.outdir}/kraken2/kraken_reports" },
```

Comment:
```suggestion
                    path: { "${params.outdir}/${params.aligner}/contaminants/kraken2/kraken_reports" },
```

Reply:
I like the idea of a contaminants directory, though I am not entirely sure if it should be tagged to the aligner rather than having its own directory like the other QC steps. I might wait on this one until one of the lead developers takes a look. We'll also have to make sure to update the `docs/output.md` file if we do make this change

Reply:
On the contrary, it to me seems that most QC tools are subsumed under the aligner? In either way, my rationale of tying it to the aligner was to make clear, that the tools depend on the unaligned fraction from that step.

```
path: { "${params.outdir}/${params.aligner}/featurecounts" },
...
path: { "${params.outdir}/${params.aligner}/dupradar/" },
...
path: { "${params.outdir}/${params.aligner}/preseq" },
...
path: { "${params.outdir}/${params.aligner}/qualimap" },
...
path: { "${params.outdir}/${params.aligner}/rseqc/bam_stat" },
...
path: { "${params.outdir}/${params.aligner}/deseq2_qc" },
```

Reply:
Oops, you're definitely right. Will do